explainer.md

```diff
@@ -39,26 +38,25 @@ The reasons for this limited scope are:
 *   Arbitrary anchors are the most basic yet useful type of anchors. There is always a need, no matter the granularity of the real world understanding the system has, to create arbitrary 3D poses as anchors. Moreover, anchors that represent arbitrary 3d poses can be used to represent positions relative to detected or tracked objects when world understanding is expanded to include object detection and tracking.
 *   Persisting and sharing anchors is outside of the current scope of this explainer, since platform-level anchors (used to implement the anchor concept) are opaque and not compatible across different platforms.
 
-Anchors are intended to maintain a pose that corresponds to a location in the physical world, and will be updated by the system as its understanding of the physical-world changes. Anchors capable of tracking moving objects are out of scope of this document.
+Anchors are intended to maintain a pose that corresponds to a location in the physical world, and will be updated by the system as its understanding of the physical world changes. Anchors capable of tracking moving objects are out of scope of this document.
```

Comment:
I think we should revisit this.  The paragraph below does not justify having moving object anchors be out of scope:  Anchors will move (by definition) and there is no clear reason why this above statement is try.

It should say "The mechanism to associate anchors with moving objects are out of scope of this document."  But we should ensure that the Anchors are designed such that they can be used by these other APIs.

Reply:
I'll remove this sentence - if/when WebXR adds velocities to the poses, this will become possible (assuming that underlying AR frameworks support moving objects).